### PR TITLE
Add default extension configuration

### LINKED ISSUE #2: Setup project to fit with chrome extension

### PR DESCRIPTION
**Pull request by HyeockJinKim**
Open react build file in extension through browser action
so that all pages can use the extension.

Closes #2

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+PUBLIC_URL=.
```
Comment (HyeockJinKim) [inline .env:1]: setting for local build file

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "name": "0Auth Extension",
+  "version": "1.0",
+  "description": "Chrome Extension for 0Auth",
+  "permissions": ["declarativeContent", "storage"],
+  "browser_action": {
+    "default_popup": "build/index.html",
+    "default_title": "0Auth",
+    "default_icon": {
+      "128": "public/icon.png"
+    }
```
Comment (HyeockJinKim) [inline manifest.json:11]: `browser_action` allows us to open the extension from all sites.

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "icons": {
+    "128": "public/icon.png"
+  },
+  "content_security_policy": "script-src 'self' 'sha256-qILRdnleFFB8nUofQbeggst+MRykHHDGiqtfJvKdtPY='",
```
Comment (HyeockJinKim) [inline manifest.json:16]: Each time we build react, we have to enter a new hash value.
This is a value to confirm that the code of the extension has not been changed.

File: public/index.html [hunk 4]
```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/icon.png" />
```
Comment (HyeockJinKim) [inline public/index.html:5]: Use our 0auth icon.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Review by anthonyminyungi — APPROVED** [review]
LGTM!

**Pull request merged**